shaky/src/main/res/menu/shaky_change_icon_test.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Similar to the previous comment, we don't need to demo this in the library module. Considering dropping it or put it in the sample app.